LEGAL.txt

```diff
@@ -0,0 +1,34 @@
+LEGAL NOTICE INFORMATION
+------------------------
+
+All the files in this distribution are covered under either the BSD 2-Clause
+License (see the file LICENSE.txt) except the files mentioned below.
+
+Library/Homebrew/vendor/mechanize/http/content_disposition_parser.rb:
```

Comment:
I wonder if just placing the license file in `Library/Homebrew/vendor/mechanize` would be sufficient?

Reply:
I think so. If anything this notice should be that we have third-party code that may use a different license in `vendor`. I doubt everything in there is `BSD-2-Clause`.

Reply:
Sounds good, I've copied the mechanize license into that folder and removed this file.